### PR TITLE
include blob upload duration in the registration requests

### PR DESCRIPTION
**Pull request by sfc-gh-gdoci**
This PR adds the file upload duration in ms and includes that information in the blob registration requests.

### CODE REVIEW

File: src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java [hunk 1]
```diff
@@ -49,6 +51,11 @@ List<ChunkMetadata> getChunks() {
     return this.chunks;
   }
 
+  @JsonProperty("upload_duration")
```
Comment (sfc-gh-tjones) [inline src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java:54]: Let's change this to `upload_duration_ms`

**Review by sfc-gh-japatel — APPROVED** [review]
lgtm! can you run formatter. https://github.com/snowflakedb/snowflake-ingest-java#code-style before pushing? (Merge gate is failing)

**Review by sfc-gh-tzhang — CHANGES_REQUESTED** [review]
I assume this is for calculating the end to end ingestion latency? If yes, we're missing two big things here:
1. The time it takes to build a blob
2. The time a blob in the registration queue, waiting be to registered

So the start timestamp we should use for each blob is when we kick off the flush job, please take a look at `flush(boolean isForce)` in `FlushService.java`

**Comment by sfc-gh-gdoci** [comment]
Duration information is added in other PRs.

**Pull request closed without merging**